packages/client/src/__tests__/types/deprecation/test.ts

```diff
@@ -30,7 +30,7 @@ async function main() {
 
   const queryRaw1 = await prisma.$queryRaw`SELECT * FROM User WHERE id = 1`
   const queryRaw2 = await prisma.$queryRaw`SELECT * FROM User WHERE id = ${1}`
-  const queryRaw3 = await prisma.$queryRaw(
+  const queryRaw3 = await prisma.$queryRawUnsafe(
```

Comment:
Interesting. I think with the new API, we'd just do:

```js
const queryRaw3 = await prisma.$queryRawUnsafe(`SELECT * FROM User WHERE id = ${2}`)
```

Instead of the prepared queries with variables. It's literally just sending this raw query string to the database.

Reply:
And of course:

```ts
await prisma.$queryRaw(`SELECT * FROM User WHERE id = ${1}`)
```

Would no longer be possible, since that's the point of this change.

Reply:
I like the word Unsafe btw :-)